src/functions/cancellations-unsubscriber/index.test.ts

```diff
@@ -0,0 +1,203 @@
+import * as dynamo from "@bods-integrated-data/shared/dynamo";
```

Comment:
Perhaps another couple of tests - one for testing the AxiosError and one for the InvalidXmlError?